WorkflowSwiftUI/Sources/ActionModel.swift

```diff
@@ -22,3 +27,15 @@ extension ActionModel: Identifiable where State: Identifiable {
         accessor.id
     }
 }
+
+#if DEBUG
+
+public extension ActionModel {
+    /// Creates a static model which ignores all sent values, suitable for static previews
+    /// or testing.
+    static func `static`(state: State) -> ActionModel<State, Action> {
```

Comment:
per @g-mark's naming thoughts – this form of utility could perhaps be named '`just`' if we want to lean into Combine's parlance

Reply:
Or follow [`Binding.constant(_:)`](https://developer.apple.com/documentation/swiftui/binding/constant(_:))'s lead.

Reply:
Sure